src/platforms/kotlin-multiplatform/index.mdx

```diff
@@ -41,3 +37,26 @@ Learn more about manually capturing an error or message in our <PlatformLink to=
 </Note>
 
 To view and resolve the recorded error, log into [sentry.io](https://sentry.io) and open your project. Clicking on the error's title will open a page where you can see detailed information and mark it as resolved.
+
+## Supported Platforms
+
+| Target Platform | Target preset                                                                                                |
+| --------------- | ------------------------------------------------------------------------------------------------------------ |
+| Android         | <ul><li>`androidTarget`</li></ul>                                                                            |
+| Kotlin/JVM      | <ul><li>`jvm`</li></ul>                                                                                      |
+| iOS             | <ul><li>`iosArm64`</li><li>`iosX64`</li><li>`iosSimulatorArm64`</li></ul>                                    |
+| macOS           | <ul><li>`macosArm64`</li><li>`macosX64`</li></ul>                                                            |
+| watchOS         | <ul><li>`watchosArm32`</li><li>`watchosArm64`</li><li>`watchosX64`</li><li>`watchosSimulatorArm64`</li></ul> |
+| tvOS            | <ul><li>`tvosArm64`</li><li>`tvosX64`</li><li>`tvosSimulatorArm64`</li></ul>                                 |
+
+## Overview of Features
+
+- Native crash reporting for Android and JVM, leveraging our [Android SDK](/platforms/android) and [Java SDK](/platforms/java)
+- Native crash reporting for iOS, macOS, tvOS, and watchOS, leveraging our [Cocoa SDK](/platforms/apple)
+- Automatic breadcrumbs for app lifecycle and UI events
+
+## Apple Installation Methods
+
+### Swift Package Manager (SPM)
+
+In addition to Cocoapods, our Kotlin Multiplatform SDK can also be installed via Swift Package Manager for Apple targets. Learn more [here](/platforms/kotlin-multiplatform/swift-package-manager/).
```

Comment:
Would it make sense to move this to after the cocoa compatibility table or under the "Install" section? It feels weird to see an alternate installation method at the very end.

Reply:
In the context of KMP using SPM is rare (it's not officially documented by Jetbrains for example) I don't have concrete numbers on hand but If I had to guess then 9/10 will not use SPM.

I'd 100% put it in the install section if we had something like tabs to switch segments on the go like we do with code snippets, or do we have those for larger segments as well?

Reply:
We only have tabs for code snippets at this point, although it's a feature we're hoping to add for the new Next.js docs site!

Based on what you've said, I agree with keeping this section where it is.